homeassistant/components/sensor/folder.py

```diff
@@ -95,10 +155,14 @@ def icon(self):
     def device_state_attributes(self):
         """Return other details about the sensor state."""
         attr = {
-            'path': self._folder_path,
-            'filter': self._filter_term,
+            CONF_FOLDER_PATH: self._folder_path,
+            CONF_FILTER: self._filter_term,
+            CONF_RECURSIVE: self._recursive,
```

Comment:
Please do not use CONF constants for attribute names.

Reply:
ok